tests/model_registry/model_registry/upgrade/test_model_registry_upgrade.py

```
@@ -1,61 +1,96 @@
     ):
-        errors = get_and_validate_registered_model(
+        validate_upgrade_model_registration(
             model_registry_client=model_registry_client[0], model_name=MODEL_NAME, registered_model=registered_model
         )
-        if errors:
-            pytest.fail("errors found in model registry response validation:\n{}".format("\n".join(errors)))
+
+    @pytest.mark.pre_upgrade
+    def test_registering_model_default_db_pre_upgrade(
+        self: Self,
+        model_registry_instance_default_db: ModelRegistry,
```

Ruff (0.14.14)

[warning] Unused method argument: `model_registry_instance_default_db`

(ARG002)